app/config/index.js

```diff
@@ -7,14 +7,20 @@ const schema = joi.object({
   env: joi.string().valid(environments.development, environments.test, environments.production).default(environments.development),
   publishPollingInterval: joi.number().default(5000),
   notifyApiKey: joi.string().allow(''),
-  notifyEmailTemplateKey: joi.string().allow('')
+  notifyEmailTemplateKey: joi.string().allow(''),
+  pactBrokerUrl: joi.string().allow(''),
+  pactBrokerUsername: joi.string().allow(''),
+  pactBrokerPassword: joi.string().allow('')
```

Comment:
No need to add these to the application itself.  The app doesn't use them, just the tests

Reply:
Agree